ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-ec.yaml

```diff
@@ -235,6 +235,7 @@ tests:
     dependency_overrides:
       OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.16
     env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
```

Comment:
Can you also apply the same change on `openshift-openshift-tests-private-release-4.14__arm64-stable.yaml` and   `openshift-openshift-tests-private-release-4.15__arm64-stable.yaml`.